types/slonik/index.d.ts

```diff
@@ -491,7 +491,10 @@ export interface ClientUserConfigurationType extends ClientConfigurationType {}
 export class SlonikError extends Error {}
 export class NotFoundError extends SlonikError {}
 export class DataIntegrityError extends SlonikError {}
-export class IntegrityConstraintViolationError extends SlonikError {}
+export class IntegrityConstraintViolationError extends SlonikError {
```

Comment:
Looks like this is missing the property `originalError: Error` which was added in this commit: https://github.com/gajus/slonik/commit/dd3af0a920118bfa393a025e15e294536a938ed1#diff-5c8b199a98cb05fcfa67ad0a331c165b

Also, there seem to be some error types missing here, namely BackendTerminatedError, ConnectionError, InvalidInputError, QueryCancelledError and UnexpectedStateError - those should probably be added too to get these types in sync with the current version of slonik.

Reply:
@mmkal check ee4f5f2